src/pudl/extract/ferc1.py

```diff
@@ -586,22 +582,24 @@ def dbf2sqlite(
     sqlite_meta.reflect(sqlite_engine)
 
     # Get the mapping of filenames to table names and fields
-    logger.info(f"Creating a new database schema based on {refyear}.")
+    logger.info(
+        f"Creating a new database schema based on {ferc1_to_sqlite_settings.refyear}.")
     datastore = Ferc1Datastore(datastore)
-    dbc_map = get_dbc_map(datastore, refyear)
+    dbc_map = get_dbc_map(datastore, ferc1_to_sqlite_settings.refyear)
     define_sqlite_db(
         sqlite_engine=sqlite_engine,
         sqlite_meta=sqlite_meta,
         dbc_map=dbc_map,
         ds=datastore,
-        tables=tables,
-        refyear=refyear,
-        bad_cols=bad_cols
+        tables=ferc1_to_sqlite_settings.tables,
+        refyear=ferc1_to_sqlite_settings.refyear,
+        bad_cols=ferc1_to_sqlite_settings.bad_cols
```

Comment:
What's your thinking on passing in these individual components, rather than the whole settings object?